src/utils.js

```diff
@@ -98,8 +98,8 @@ const isValidCursor = (cursor, order) => {
 
 const recursivelyGetPaginationQuery = (order, cursor) => {
   const directionValueIndex = order[0].length - 1;
-  const currentOp =
-    order[0][directionValueIndex].toLowerCase().split(' ')[0] === 'desc' ? Op.lt : Op.gt;
+  // if cursor[0] is null then we set a not equal operator
+  const currentOp = cursor[0] === null ? Op.ne : order[0][directionValueIndex].toLowerCase().split(' ')[0] === 'desc' ? Op.lt : Op.gt;
```

Comment:
we should not add default as `ne`

Reply:
@ShreyazSathish lets add test....

Reply:
added tests in https://github.com/goSprinto/sequelize-cursor-pagination/pull/2/commits/c451a59b52f340611fa88c0f37f4dd08e37b84a5

Reply:
Update description to explain the `ne` part.